<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Forecast_Tab2_HarvestTab</class>
 <widget class="QWidget" name="Forecast_Tab2_HarvestTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>552</width>
    <height>259</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="Forecast_Tab2_HarvestGB">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="title">
      <string>Forecast Harvest Data</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="Forecast_Tab2_SetHarvestTypePB">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>55</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>55</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Toggles between loading Effort and Catch data</string>
          </property>
          <property name="statusTip">
           <string>Toggles between loading Effort and Catch data</string>
          </property>
          <property name="whatsThis">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Harvest Type&lt;/span&gt;&lt;/p&gt;&lt;p&gt;This button toggles between loading Effort and Catch data. Either can be used in generating forecasts.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>Effort:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="toolTip">
           <string>Enable multiplier widgets</string>
          </property>
          <property name="statusTip">
           <string>Enable multiplier widgets</string>
          </property>
          <property name="whatsThis">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Multiplier&lt;/span&gt;&lt;/p&gt;&lt;p&gt;The Multiplier widgets allow the user to modify the selected Harvest data based upon the value given and the type of modfication selected. A modification type of Constant sets all rows to the product of the first row and the multiplier. A modification type of Variable sets all rows to the product of the previous row and the multiplier.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>Multiplier:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="Forecast_Tab2_MultiplierCB">
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="toolTip">
           <string>Enable multiplier widgets</string>
          </property>
          <property name="statusTip">
           <string>Enable multiplier widgets</string>
          </property>
          <property name="whatsThis">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Multiplier&lt;/span&gt;&lt;/p&gt;&lt;p&gt;The Multiplier widgets allow the user to modify the selected Harvest data based upon the value given and the type of modfication selected. A modification type of Constant sets all rows to the product of the first row and the multiplier. A modification type of Variable sets all rows to the product of the previous row and the multiplier.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="Forecast_Tab2_MultiplierDSB">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="toolTip">
           <string>The value by which to multiply data in the Harvest table. (Works on selected cells.)</string>
          </property>
          <property name="statusTip">
           <string>The value by which to multiply data in the Harvest table. (Works on selected cells.)</string>
          </property>
          <property name="whatsThis">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Multiplier&lt;/span&gt;&lt;/p&gt;&lt;p&gt;The Multiplier widgets allow the user to modify the selected Harvest data based upon the value given and the type of modfication selected. A modification type of Constant sets all selected rows to the product of the first row and the multiplier. A modification type of Variable sets all selected rows to the product of the previous row and the multiplier.&lt;/p&gt;&lt;p&gt;N.B. This option works on selected rows and cells.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="decimals">
           <number>1</number>
          </property>
          <property name="minimum">
           <double>0.100000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
          <property name="value">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="Forecast_Tab2_MultiplierCMB">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>85</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>85</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="toolTip">
           <string>Constant = Set rows to product of multiplier and first row.
Variable = Set rows to product of previous row and multiplier.</string>
          </property>
          <property name="statusTip">
           <string>Constant = Set rows to product of multiplier and first row.  Variable = Set rows to product of previous row and multiplier.</string>
          </property>
          <property name="whatsThis">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Multiplier&lt;/span&gt;&lt;/p&gt;&lt;p&gt;The Multiplier widgets allow the user to modify the selected Harvest data based upon the value given and the type of modfication selected. A modification type of Constant sets all selected rows to the product of the first row and the multiplier. A modification type of Variable sets all selected rows to the product of the previous row and the multiplier.&lt;/p&gt;&lt;p&gt;N.B. This option works on selected rows and cells.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <item>
           <property name="text">
            <string>Constant</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Variable</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Forecast_Tab2_SetDefaultHarvestPB">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="toolTip">
           <string>Sets the table to the last row of the Estimation Harvest table</string>
          </property>
          <property name="statusTip">
           <string>Sets the table to the last row of the Estimation Harvest table</string>
          </property>
          <property name="whatsThis">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Default Set&lt;/span&gt;&lt;/p&gt;&lt;p&gt;This button copies the last row of the Harvest Estimation table to every row in the following table.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>Set</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="whatsThis">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Catch Units&lt;/span&gt;&lt;/p&gt;&lt;p&gt;The user may enter data in one of the following formats: &lt;/p&gt;&lt;ul style=&quot;margin-top: 0px; margin-bottom: 0px; margin-left: 0px; margin-right: 0px; -qt-list-indent: 1;&quot;&gt;&lt;li style=&quot; margin-top:12px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;lbs (pounds) &lt;/li&gt;&lt;li style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;kg (kilograms) &lt;/li&gt;&lt;li style=&quot; margin-top:0px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;mt (metric tons)&lt;/li&gt;&lt;/ul&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>Units:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="Forecast_Tab2_UnitsCMB">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>45</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>45</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="toolTip">
           <string>Units for table below</string>
          </property>
          <property name="statusTip">
           <string>Units for table below</string>
          </property>
          <property name="whatsThis">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Forecast Harvest Units&lt;/span&gt;&lt;/p&gt;&lt;p&gt;The user may enter data in one of the following formats: &lt;/p&gt;&lt;ul style=&quot;margin-top: 0px; margin-bottom: 0px; margin-left: 0px; margin-right: 0px; -qt-list-indent: 1;&quot;&gt;&lt;li style=&quot; margin-top:12px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;lbs (pounds) &lt;/li&gt;&lt;li style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;kg (kilograms) &lt;/li&gt;&lt;li style=&quot; margin-top:0px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;mt (metric tons)&lt;/li&gt;&lt;/ul&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="currentIndex">
           <number>2</number>
          </property>
          <item>
           <property name="text">
            <string>lbs</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>kg</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>mt</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="Forecast_Tab2_ConvertCB">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="toolTip">
           <string>Convert data when checked</string>
          </property>
          <property name="statusTip">
           <string>Convert data when checked</string>
          </property>
          <property name="whatsThis">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Convert Checkbox&lt;/span&gt;&lt;/p&gt;&lt;p&gt;If this box is checked, then when the user changes the units pulldown, the data below will be converted. &lt;/p&gt;&lt;p&gt;If this box is not checked, then the units pulldown may be changed with no conversion applied to the data.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableView" name="Forecast_Tab2_HarvestTV">
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="statusTip">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_88">
     <item>
      <widget class="QPushButton" name="Forecast_Tab2_PrevPB">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Go to the previous page</string>
       </property>
       <property name="statusTip">
        <string>Go to the previous page</string>
       </property>
       <property name="text">
        <string>&lt;-</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_100">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="Forecast_Tab2_LoadPB">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="toolTip">
        <string>Load table data</string>
       </property>
       <property name="statusTip">
        <string>Load table data</string>
       </property>
       <property name="text">
        <string>Load</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Forecast_Tab2_SavePB">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="toolTip">
        <string>Save above table data</string>
       </property>
       <property name="statusTip">
        <string>Save above table data</string>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="Forecast_Tab2_AutoSaveCB">
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="toolTip">
        <string>Saves the Harvest data after each edit. Then runs the Forecast.</string>
       </property>
       <property name="statusTip">
        <string>Saves the Harvest data after each edit. Then runs the Forecast.</string>
       </property>
       <property name="whatsThis">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Auto Save&lt;/span&gt;&lt;/p&gt;&lt;p&gt;Checking this box will save the Harvest data immediately after each edit.  The Forecast will then run and the plot will be updated.  This feature allows the user to quickly see the result of their Harvest modification.&lt;/p&gt;&lt;p&gt;N.B. Because a database save and chart update are being run in real-time, this feature may appear sluggish on slower computer systems.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_46">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>48</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="Forecast_Tab2_NextPB">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Go to the next tab page</string>
       </property>
       <property name="statusTip">
        <string>Go to the next tab page</string>
       </property>
       <property name="text">
        <string>-&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
